ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/blocks/blockbody/BeaconBlockBody.java

```diff
@@ -75,6 +76,8 @@ default Optional<ExecutionPayloadSummary> getOptionalExecutionPayloadSummary() {
     return Optional.empty();
   }
 
+  Optional<SszList<SignedBlsToExecutionChange>> getOptionalBlsToExecutionChanges();
```

Comment:
why don't you go for
```java
  default Optional<SszList<SignedBlsToExecutionChange>> getOptionalBlsToExecutionChanges() {
    return Optional.empty();
  }
 ````
 like the other `getOptional`?

Reply:
Adrian has strong views on it being an anti-pattern, so I tried to avoid it...

Reply:
Well strong is probably over stating it, but if the expectation is that any new implementations would want to implement the method, then it's probably better to not have a no-op default implementation - that way you can't forget to implement it.  Default methods do make a lot of sense if they're likely to be the right implementation for new classes though.